src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
```

Comment:
When you just want to output a single element, you just need to use its index. Specifying a range will always output a list.